src/SprykerSdk/Yves/Benchmark/Request/RequestBuilder.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Yves\Benchmark\Request;
+
+use GuzzleHttp\Psr7\Request;
+use Psr\Http\Message\RequestInterface;
+use SprykerSdk\Shared\Benchmark\Exception\HttpMethodNotAllowed;
+use SprykerSdk\Shared\Benchmark\Request\RequestBuilderInterface;
+use SprykerSdk\Yves\Benchmark\Dependency\Service\BenchmarkToUtilEncodingServiceInterface;
+use SprykerSdk\Yves\Benchmark\BenchmarkConfig;
+
+class RequestBuilder implements RequestBuilderInterface
```

Comment:
Couldn't you extract Shared code? IMO this code is duplicated among the applications.